README.md

```diff
@@ -353,6 +353,41 @@ Releasing
 
 [How to Release Apache Fineract](https://cwiki.apache.org/confluence/x/DRwIB) documents the process how we make the source code that is available here in this Git repository into a binary release ZIP available on http://fineract.apache.org.
 
+Before you use Gradle to create a release you need to make sure that your GPG is properly setup and that you have set the following properties in your ~/gradle/gradle.properties:
+```
+signing.gnupg.keyName=7890ABCD
+signing.gnupg.passphrase=secret
+```
+
+IMPORTANT: Do not set your GPG secrets in one of the project gradle.properties and double check that you are not accidentally committing them to Git.
+
```

Comment:
To avoid exactly this, is it possible to instruct people instead to pass these as command line parameters when they run the build?

Reply:
You can do this anyway. Personally I prefer to set this up with my personal gradle.properties in my home folder, but it's not required; you can just pass the required parameters with "-P" on the command line and it'll work as well.

Reply:
Thanks. I know you can do this either way, I just wondered if it would be better in the README to suggest that way - rather than suggest that people put it in gradle.properties AND warn them not to check it in. If the suggested way is to use command line there's no risk that people will check anything in.

Reply:
I've added a clarification to the README describing both ways to provide the necessary parameters (either in your home gradle.properties or via command line).

Reply:
This section is anyway only intended for a limited audience (I think there are 3 or 4 people right now that can sign the releases). But you have a point here. Happy to change the default recommendation for the next release. Maybe you can raise that issue then on the mailing list to see what others have to contribute on that subject?